it/mail/chrome/messenger/search.properties

```diff
@@ -28,12 +28,12 @@ moreButtonTooltipText=Aggiungi regola
 lessButtonTooltipText=Elimina questa regola
 
 # pbMessageSearch.js
-threadPaneNoMatchesSuggestionWildcard=Try appending the wildcard to your search terms, like %1$S.
+threadPaneNoMatchesSuggestionWildcard=Prova ad aggiugnere la wildcard ai tuoi termini di ricerca, come %1$S.
```

Comment:
CORRECTION NEEDED. 

There is a typo. Please replace "aggiugnere" with "aggiungere".